### DISCUSSION TITLE
I cannot find the .bin file for the latest release

### DISCUSSION THREAD

**Discussion by JoeMaxey** [mriscoc/Ender3V2S1#1227] [General]
I've downloaded the Source Code from the assets portion at the bottom of the screen and opened the zip file. However, I cannot find the bin file to update the printer with. Am I missing something?

**Comment by JamieLee2k** [comment]
Yeah I was thinking the same thing, I found the source code though

**Comment by mriscoc** [comment]
Unrestricted binaries for the 20240122 version will be released only after testing.
